src/panels/lovelace/cards/types.ts

```diff
@@ -309,7 +310,7 @@ export interface StatisticsGraphCardConfig extends LovelaceCardConfig {
   unit?: string;
   days_to_show?: number;
   period?: "5minute" | "hour" | "day" | "month";
-  stat_types?: StatisticType | StatisticType[];
+  stat_types?: ExtendedStatisticType | ExtendedStatisticType[];
```

Comment:
I'm not sure why the type checker didn't detect that this could be `change` as well when configuring?